test/unit/UsersTest.php

```diff
@@ -118,21 +118,6 @@ function testGetUserByGuidsEmptyTokenThrowsException()
         $personaClient->getUserByGuids(['123'], '');
     }
 
-    function testGetUserByGuidsInvalidTokenThrowsException()
```

Comment:
Not a unit test - talks to the network without request/response stubbing. Moved to integration test suite.